gdc_client/query/versions.py

```diff
@@ -29,6 +30,16 @@ def get_latest_versions(url, uuids, verify=True):
     for chunk in _chunk_list(uuids):
         resp = requests.post(versions_url, json={"ids": chunk}, verify=verify)
 
+        if not resp.ok:
+            raise ServerError(
```

Comment:
One thing is now that you've split this into Client/Server exceptions/errors this isn't always the correct logic. The 400s are client errors and only the 500s are server errors. But because ok is >=400 you are throwing the same for each.

Reply:
I was looking for a generic name for an exception that resulted from this failed http request in general. I can change it to something like HTTPError or something similar.